clients/spark/core/src/test/resources/parser-test/sst_files_generator.go

```diff
@@ -90,15 +90,30 @@ func newGenerateNanoid(size int) func() (string, error) {
 }
 
 func newGenerateFuzz() func() (string, error) {
-	return newGenerateFuzzWithSharedPrefixes(EmptyPrefix)
+	f := fuzz.NewWithSeed(FuzzerSeed)
+	return func() (string, error) {
+		var val string
+		f.Fuzz(&val)
+		return val, nil
+	}
 }
 
-func newGenerateFuzzWithSharedPrefixes(sharedPrefix string) func() (string, error) {
+func newGenerateFuzzWithSharedPrefixes() func() (string, error) {
 	f := fuzz.NewWithSeed(FuzzerSeed)
+	src := rand.NewSource(041021)
+	r := rand.New(src)
+	prefixBaseLen := 1 + r.Intn(100)
+	sharedPrefixBase, err := nanoid.New(prefixBaseLen)
```

Comment:
This is pure-ASCII.  Previously I think we had actually binary junk; that could be better because it might detect places where we performed bad binary -> String conversions.